python/osrd_schemas/osrd_schemas/infra_editor.py

```diff
@@ -159,14 +159,73 @@ class _TmpSignal(BaseModel):
 
 
 if __name__ == "__main__":
+    import argparse
+    import re
     from json import dumps
 
+    parser = argparse.ArgumentParser(
+        description="Generate the infra editor railjson schema."
+    )
+    parser.add_argument(
+        "--translation",
+        action="store_true",
+        help="Generate the infra editor English translation json instead.",
+    )
+    args = parser.parse_args()
+
     railjson_schema = infra.RailJsonInfra.model_json_schema()
     tmp_signal_schema = _TmpSignal.model_json_schema()
     railjson_schema["$defs"].update(tmp_signal_schema["$defs"])
     railjson_schema["$defs"]["Signal"]["properties"].update(
         tmp_signal_schema["properties"]
     )
 
-    # sort keys in order to diff correctly in the CI
-    print(dumps(railjson_schema, indent=4, sort_keys=True))
+    if not args.translation:
+        # sort keys in order to diff correctly in the CI
+        print(dumps(railjson_schema, indent=4, sort_keys=True))
+
+    else:
+
+        def normalize_title_case(title: str):
```

Comment:
I must admit, I am not a fan in principle of auto-generating human-readable `title` properties from code. That said, it sounds like fixing this would require quite a bit of effort, and we'd like to switch to a Rust-generated RailJSON schema at some point in the future. So let's stick with that for now!